spec/lib/audit/checksum_spec.rb

```diff
@@ -13,87 +13,32 @@
 
   describe '.validate_disk' do
     include_context 'fixture moabs in db'
-    let(:subject) { described_class.validate_disk(endpoint_name, limit) }
 
-    context 'when there are PreservedCopies to check' do
-      let(:cv_mock) { instance_double(ChecksumValidator) }
-
-      it 'creates an instance and calls #validate_checksums for every result when results are in a single batch' do
-        allow(ChecksumValidator).to receive(:new).and_return(cv_mock)
-        expect(cv_mock).to receive(:validate_checksums).exactly(3).times
-        described_class.validate_disk(endpoint_name, limit)
-      end
-
-      it 'creates an instance and calls #validate_checksums on everything in batches' do
-        pcs_from_scope = PreservedCopy.by_endpoint_name(endpoint_name).fixity_check_expired
-        cv_list = pcs_from_scope.map { |pc| ChecksumValidator.new(pc) }
-        expect(cv_list.size).to eq 3
-        cv_list.each do |cv|
-          allow(ChecksumValidator).to receive(:new).with(cv.preserved_copy).and_return(cv)
-          expect(cv).to receive(:validate_checksums).exactly(1).times.and_call_original
-        end
-        described_class.validate_disk(endpoint_name, 2)
-      end
```

Comment:
i would prefer to keep the old style of test that makes sure that each of the pres copies we care about was examined, not just that the CV call was made the right number of times.  i would worry less about this if we switched to `find_each`, since it feels less necessary to test ActiveRecord's batching mechanism than it does to test our own home-grown batching mechanism, with it's less-than-obvious usage guidelines.

Reply:
Switched to `find_each`, so I think all is good.